src/app/frontend/deploy/deploy_controller.js

```diff
@@ -115,7 +128,7 @@ export default class DeployController {
    * @export
    */
   isDeployDisabled() {
-    return this.isDeployInProgress_;
+    return this.isDeployInProgress_ || this.deployLabelService_.hasDuplicationError(this.labels);
```

Comment:
I can check if this will work. This depends if this custom validation based on label property changes form validation state.